packages/material/scss/scrollview/_theme.scss

```diff
@@ -0,0 +1,28 @@
+$scrollview-navigation-color: $base-text;
+
+@import "~@progress/kendo-theme-default/scss/scrollview/theme";
+
+@include exports("scrollview/theme/material") {
+    $scrollview-text-opacity: .4 !default;
+    $scrollview-bottom: $scrollview-pager-spacing-x * 2;
+
+    .k-scrollview-next:hover,
+    .k-scrollview-prev:hover {
+        &::before {
+            background-color: rgba($base-bg, $scrollview-text-opacity);
+        }
+    }
+
+    .k-scrollview-pageable {
+        bottom: $scrollview-bottom;
+
+        li.k-button {
+            background-color: rgba(black, $scrollview-text-opacity);
+            margin: $scrollview-pager-spacing-y $scrollview-pager-spacing-x;
```

Comment:
The same goes for this `margin`. There is a margin declaration in theme-default, which can use these variables.